app/src/main/java/org/mozilla/fenix/settings/advanced/LocaleViewHolders.kt

```diff
@@ -20,8 +20,9 @@ class LocaleViewHolder(
 ) : BaseLocaleViewHolder(view, selectedLocale) {
 
     override fun bind(locale: Locale) {
-        // capitalisation is done using the rules of the appropriate locale (endonym and exonym)
+        // Capitalisation is done using the rules of the appropriate locale (endonym and exonym).
```

Comment:
ironic that we're capitalizing capitalisation

Reply:
😎